core/src/main/java/org/frankframework/cache/AbstractCacheAdapter.java

```diff
@@ -62,9 +62,9 @@ public abstract class AbstractCacheAdapter<V> implements ICache<String,V>, Frank
 	private TransformerPool valueTp=null;
 
 	@Override
-	public void configure(String ownerName) throws ConfigurationException {
+	public void configure() throws ConfigurationException {
 		if (StringUtils.isEmpty(getName())) {
-			setName(ownerName+"_cache");
+			setName(applicationContext.getId()+"_cache");
```

Comment:
This is the adapter name, which was previously set in the pipeline (and passed through in configure).